releasenotes/notes/remove-deprecated-circuit-methods-afc22d0ba59ac853.yaml

```
@@ -0,0 +1,11 @@
+---
+deprecations:
```

Oops missed this before:

```suggestion
upgrade:
```